src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -270,10 +272,10 @@ def main():
     if source not in SOURCE:
         raise ValueError(f'Unexpected source. Currently supported sources: {SOURCE}')
 
-    work_dir = _make_work_dir(root_dir, SOURCE_DATA_DIR, source, release_ver)
+    work_dir = _make_work_dir(root_dir, loader_common_names.SOURCE_DATA_DIR, source, release_ver)
 
     genome_ids = _fetch_genome_ids(source, release_ver, work_dir)
-
+    genome_ids = genome_ids[:10]
```

Comment:
Sorry, I'm confused here. Why are we skipping the first 10 ids?

Reply:
removed. I was running a test and forget to remove it.

Reply:
:+1: